### PR TITLE
Resolve rename

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: modules/default.nix [hunk 1]
```diff
@@ -1,5 +1,8 @@
 { lib, pkgs, ... }:
 
+let
+  colorizer = if lib.versionAtLeast lib.version "25.05pre" then "colorizer" else "nvim-colorizer";
```
Comment (MarcelCoding) [inline modules/default.nix:4]: Are there guarantees that people are not using the default branch of nixvim with the stable version of nixpkgs?

Comment (MarcelCoding) [reply modules/default.nix:4]: It did it myself until very recently because nixvim did not yet branch off back then.

File: README.md [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
   home-manager = {
-    url = "github:nix-community/home-manager/release-24.05";
+    url = "github:nix-community/home-manager/release-24.11";
     inputs.nixpkgs.follows = "nixpkgs";
   };
-  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.11";
   nixvim = {
-    url = "github:nix-community/nixvim/nixos-24.05";
+    url = "github:nix-community/nixvim/nixos-24.11";
```
Comment (MarcelCoding) [inline README.md:14]: Shouldn't these be all pointing to unstable? Ci is testing stable using override inputs.

File: modules/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply modules/default.nix:4]: I think people should be using the matching nixpkgs version otherwise packages might not match and be missing.

**Review by MarcelCoding — APPROVED** [review]

**Pull request merged**